src/DotNetClient/DotNetClient/Wonde/Helpers/JsonDictionaryConverter.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Wonde.Helpers
+{
+    internal class JsonDictionaryConverter : JsonConverter<Dictionary<string, object>>
```

Comment:
You were using a class called JavascriptSerializer which is dotnet framework exclusive.

I switched out this serializer for System.Text.Json.Serializer instead.

JavascriptSerializer deserialized objects into Dictionary<string, object>
But System.Text.Json.Serializer deserialized into JObjects, and thus cannot be casted to Dictionary<string,object>.

To add a compatibility layer, I created this converter that can be plugged into the serializer for automatic conversion.